modules/programs/git.nix

```diff
@@ -433,7 +433,7 @@ in {
         "git/config".text = generators.toGitINI cfg.iniContent;
 
         "git/ignore" = mkIf (cfg.ignores != [ ]) {
-          text = concatStringsSep "\n" cfg.ignores + "\n";
+          text = concatMapStringsSep "\n" (x: "include ${x}") cfg.ignores;
```

Comment:
@ambroisie do you mean like adding a `mkBefore` ?

Reply:
Yeah `mkBefore`/`mkAfter` is what I had in mind. Not sure which is better, presumably `mkAfter` for local-only changes (would allow overrides _I assume_).

Reply:
Also, I'm assuming the `git` change is unintentional?

Reply:
should all be fixed.